### ISSUE TITLE
[Feature] Speech to text on the Search Bar.

### ISSUE THREAD

**Issue by The-cpu-max** [jurialmunkey/skin.arctic.horizon.2#944]
### Skin section

Other

### Feature description

The voice search option that exists in Android is not good for operation
I will ask you to add a shortcut
A button like it would be in Google PLAY if you know
If you have an Android TV at home, you probably know how inconvenient it is

### Why is this feature needed? Please justify and explain.

Adding the abbreviation is also mandatory on the keyboard
And also on the main page above
Lots of people need it, and just don't know how to explain it correctly

### Screenshots and Additional Info

![183499467-d03cbb91-9ab3-4bee-9126-22511c7d043a](https://github.com/jurialmunkey/skin.arctic.horizon.2/assets/102416233/c835b5e1-1641-4c84-8af5-4626f60df4fe)
![183499704-5007af19-8ae7-4bfc-823e-da71a3e922cc](https://github.com/jurialmunkey/skin.arctic.horizon.2/assets/102416233/62d74b1c-7c47-4a57-bff4-bf8c1766d470)

### Checklist

- [X] I have made a separate issue for each feature I am requesting.
- [X] I have searched the issues and this feature has not been requested.
- [X] I am using the most recent skin version.

**Comment by jurialmunkey** [comment]
Are there any skins which implement this? I'm pretty sure it's not possible unless there is a feature I'm not aware of.

**Comment by The-cpu-max** [comment]
> 
It does exist in several skins
All you have to do is simple
Just create a shortcut
The function itself is already embedded in Android
Let me know if there is a problem adding the feature
If so, I will try to get me a link to another skin

**Comment by jurialmunkey** [comment]
What skin?

**Comment by The-cpu-max** [comment]
> What skin?

https://github.com/kodi7rd/skin.fentastic
on xml/DialogKeyboard.xml
"Voice Search Button"

**Comment by jurialmunkey** [comment]
Okay, interesting thanks. That `VoiceRecognizer` command doesn't seem to be documented anywhere but it looks like it is in the official keymap so I guess it should work and I should be able to add a button.

>  The RTL problem of the search does not exist there

That's because it does not use an Edit control which allows typing directly into the box. The issue is due to how Kodi treats RTL languages in the edit control. Until it is fixed in Kodi, there is no way to allow dynamically typed search.

**Comment by rowel07** [comment]
I think its already there. Just goto the text bar of the keyboard then press and hold the center button of your remote. Feature depends also on the device you are using....

**Comment by jurialmunkey** [comment]
> I think its already there. Just goto the text bar of the keyboard then press and hold the center button of your remote. Feature depends also on the device you are using....

Yeah default keymap sets long press select on keyboard to activate voice search. Though I can understand how a dedicated button on the keyboard would be useful for users who might not be aware that you can do that.
